pyron-core/src/main/scala/com/cloudentity/pyron/domain/flow/flow.scala

```diff
@@ -93,16 +92,19 @@ object TargetService {
     DiscoverableService(serviceName)
 
   private def readStaticService(req: HttpServerRequest): StaticService = {
+    val ssl = req.isSSL
     if (Option(req.host()).isDefined) {
-      Option(req.host()).get.split(":").toList match {
-        case h :: Nil =>      StaticService(TargetHost(h), 80, req.isSSL())
```

Comment:
I think we should leave the formatting as is, or if we're deciding to change it it should be done in one PR for the whole repo.

Reply:
it's more about using char instead of string in the split. I think it's always better to use char when splitting on single char, because it's not tried as regex pattern. Other lines changed because of dropping repeated call to isSSL() so ... I would say formatting change is a by-product :)

Reply:
I'm referring to the indentation of the right hand of the pattern match.
From what I see it's aligned on all lines in other places, let's either keep it as is or change it everywhere in another PR.